packages/agent-sdk/src/evm/index.ts

```diff
@@ -59,25 +59,18 @@ export async function validateRequest<
 >(
   req: TRequest,
   // TODO(bh2smith): Use Bitte Wallet's safeSaltNonce as Default.
-  safeSaltNonce: string,
   responder?: (data: object, init?: { status?: number }) => TResponse,
 ): Promise<TResponse | null> {
   const createResponse = responder ? responder : fallbackResponder;
   const metadataHeader = req.headers.get("mb-metadata");
   const metadata = JSON.parse(metadataHeader ?? "{}");
   const { accountId, evmAddress } = metadata;
-  if (!accountId || !evmAddress) {
+  if (!accountId && !evmAddress) {
```

Comment:
I think we should still be erroring if only one is provided? Why was this change included here?

Reply:
This can be refined but we don't need to have both to create evm transactions anymore